content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md

```diff
@@ -37,7 +37,7 @@ if your cluster is running v1.16 then you can use kubectl v1.15, v1.16
 or v1.17; other combinations
 [aren't supported](/docs/setup/release/version-skew-policy/#kubectl).
 
-On each of the Node(s) you're reconfiguring, the Kubelet's `--dynamic-config-dir`
+On each of the Node(s) you're reconfiguring, the kubelet's `--dynamic-config-dir`
```

Comment:
Line 40, lowercase node?
For each node that you're reconfiguring, you must set the kubelet `--dynamic-config-dir` flag to a writable directory.